serverless-configs/aws-functions.yml

```diff
@@ -55,3 +65,72 @@ gkrSendFeedback:
         cors: ${file(./serverless-configs/${self:provider.name}-cors-configuration.yml)}
```

Comment:
I think we'll eventually want some sort of authentication (edlAuthorizer) on this GKR lambdas, as access should be protected to just MMT, especially since this was updates the machine learning model.